### PR TITLE
🧚‍♀️Project 3 final touches 

### PR DESCRIPTION
**Pull request by ocrearyy**
In this project I did the following:

- Ensure to have a fully working app that uses only basic styles to make the layout work⚒✔
- Improve the look and feel of the application, adding the styles of your choice💯✔
- Keep the general layout of the wireframe🕸✔
- Use plain CSS👨‍💻✔

### CODE REVIEW

File: src/modules/LeaderboardAPI.js [hunk 1]
```diff
@@ -41,9 +41,7 @@ const showScores = async (userData) => {
   scoresDisplay.innerHTML = '';
   sortedData.forEach((element) => {
     const score = `
-    <div>
-    <p>${element.user}:<span>${element.score}</span></p>
-    </div>
+    <li>${element.user}:<span class="space">${element.score}</span></li>
     `;
     scoresDisplay.innerHTML += score;
```
Comment (micheaol) [inline src/modules/LeaderboardAPI.js:46]: Please ensure that the app style and its layout look like the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png).

Your Implementation | wireframe Design
:---:|:---:
image | image

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @oshanedesign , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/modules/LeaderboardAPI.js (hunk 1 above)
Comment (ocrearyy) [reply src/modules/LeaderboardAPI.js:46]: @micheaol thank you, app style and layout updated. 
e45

**Review by aimemalaika — APPROVED** [review]
Hi @oshanedesign , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**